_posts/RhymesWithMecca/2016-02-09-BeccaandJames'Chapter8Part2.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+author:
+-RhymesWithMecca
+-jamesma560
+title: Becca and James' Chapter 8 Exercises, Part 2
+---
+
+**Here is 8.4:**
+<iframe src="https://trinket.io/embed/python/c460c7b95a" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Here is 8.5:**
+<iframe src="https://trinket.io/embed/python/bed31f5373" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Here is 8.6:**
+<iframe src="https://trinket.io/embed/python/6913c1e4af" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+**Reflection (do we need one?  Here it is regardless:**
+Paired programming is useful.  We were definitely faster with two brains than I would have been with one.  I am trying to make my 
+markdown fancy for the first time.  Hope it works!
```

Comment:
Nice job!  Bonus karma points for the reflection.  There will be plenty of reflecting going on in your mid-semester reflection.